src/obfl-specification.html

```diff
@@ -1167,7 +1167,7 @@ <h4 id="L1023" class="include">block</h4>
       </dl>
     </dd>
   <dt>Content Model</dt>
-    <dd>text(), leader, marker, br, evaluate, page-number, marker-reference,
+    <dd>text(), leader, marker, br, evaluate, page-number, blockmeta, marker-reference,
```

Comment:
`blockmeta`? I would expect `external-reference`

Reply:
Well, I forgot to update the specification. Thanks for notifying me.